congratulations/Congratulations.py

```diff
@@ -0,0 +1,52 @@
+from PIL import ImageTk, Image
+import webbrowser as webbrowser
+from urllib.parse import quote
+try:
+    import Tkinter as Tk        # Python < 3.0
+except ImportError:
+    import tkinter as Tk        # Python >= 3.0
+
+class Congratulations(object):
+    """Represents the completion of the goal. Houses functionality
+    for displaying a congratulatory message and prompting the user
+    for further action.
+
+    TO DO:
+        Allow the user to start up a new goal.    
+    """
+
+    def __init__(self):
+        """Open a Tkinter window with a congratulatory message.
+        """
+        # Attributes
+        self.root = Tk.Toplevel()
+        self.root.title("Congratulations!")
+        self.challenge_duration = 100        # Set this manually for now. Pass it in as an argument later.
+
+        CANVAS_WIDTH = 300
+        CANVAS_HEIGHT = 200
+        self.canvas = Tk.Canvas(self.root, width = CANVAS_WIDTH, height = CANVAS_HEIGHT)
+        self.canvas.pack()
+
+        # Text label
+        lbl = Tk.Label(self.root, text="Congratulations!\nYou've reached the end of your challenge! You've earned the right to boast a little, so why not tell everyone about your accomplishment?",
```

Comment:
Just out of curiosity, how come you are using labels, rather than just a create_text object? Just trying to learn new (and potentially smarter ways of doing things - I've been using create_text all along)

Reply:
Honestly, I have no good answer for this, heh. I'm used to working with labels in Windows forms, and it seemed like a more natural way of doing things. But I have very little experience with Tkinter and so I really can't say what benefits one method might have over the other.

Reply:
That's fair. I'll try and do some research into this myself. I'll let you know what I find :)